packages/components/src/components/spinner/spinner.stories.ts

```diff
@@ -1,75 +1,69 @@
 import '../../solid-components';
 import { html } from 'lit-html';
-import { storybookDefaults, storybookTemplate } from '../../../scripts/storybook/helper';
+import { storybookDefaults } from '../../../scripts/storybook/helper';
 import { withActions } from '@storybook/addon-actions/decorator';
 
 const { argTypes, args, parameters } = storybookDefaults('sd-spinner');
-const { generateTemplate } = storybookTemplate('sd-spinner');
+
+/**
+ * Visual indicator with a looping animation that shows loading is in process.
+ */
 
 export default {
   title: 'Components/sd-spinner',
   component: 'sd-spinner',
+  tags: ['!dev'],
   args,
   argTypes,
   parameters: { ...parameters },
   decorators: [withActions] as any
 };
 
-/**
- * Default: This shows sd-spinner in its default state.
- */
-
 export const Default = {
-  render: (args: any) => {
-    return generateTemplate({ args });
-  }
+  name: 'Default',
+  render: () => html`<sd-spinner></sd-spinner>`
 };
 
 /**
- * Use the `color` attribute to change the color of the spinner and correspond (currentColor) with the parents color.
+ * Offers 4 `color` variants for alternate experiences:
```

Comment:
Offers “color” variants for alternate experiences:

Reply:
Done